Runtime/Scripts/EdgeEventsManager.cs

```diff
@@ -616,6 +616,7 @@ void ProcessLatency(Statistics stats)
       else
       {
         Logger.Log("Current Latency (" + stats.Avg + ") < LatencyThreshold (" + config.latencyThresholdTriggerMs + ")");
+        processingStatus = LatencyProcessingStatus.Ready;
```

Comment:
Ok. I assume nothing is waiting on this status, just for read only whatever wants to start the next test.